syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -943,7 +943,7 @@ def max(self, dim=None, return_idx=False):
             the maximum value (possibly across an axis)
             and optionally the index of the maximum value (possibly across an axis)
         """
-        values = self
+        values = self.copy()
```

Comment:
Does this inherently require copying because operations later in this function are in-place? Could it be avoided by changing the lines below that read something like `values = values.some_op()`?

Reply:
Removed (redundant operation)